frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/works/useSearch.js

```diff
@@ -1,16 +1,10 @@
 import React from "react";
-import { useQuery } from "react-query";
+import { useMutation } from "react-query";
 import { WorksService } from "../../services/elements/Works";
 
 
-const useSearch = ({ tenantId, filters, config = {} }) => useQuery(
```

Comment:
@burhanj-eGov Don't change the hooks that I have created. Make a new file for your requirement